micropip/package_index.py

```diff
@@ -9,11 +9,11 @@
 from typing import Any
 from urllib.parse import urljoin, urlparse
 
+from ._cached_version import CachedVersion, InvalidVersion, parse_cached_version
```

Comment:
Was there some reason to import and re-export InvalidVersion inside the _cached version? I think we can keep importing it from the packaging module as-is.